app/src/vanilla/java/cm/aptoide/pt/app/view/EditorialFragment.java

```diff
@@ -363,8 +402,123 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     ready.onNext(null);
   }
 
-  @Override public Observable<String> mediaContentClicked() {
-    return editorialMediaClicked;
+  @Override public Observable<ScrollEvent> placeHolderVisibilityChange() {
+    return RxNestedScrollView.scrollChangeEvents(scrollView)
+        .flatMap(viewScrollChangeEvent -> Observable.just(viewScrollChangeEvent)
+            .map(scrollDown -> isItemShown())
+            .distinctUntilChanged()
+            .map(isItemShown -> new ScrollEvent(
+                isScrollDown(viewScrollChangeEvent.oldScrollY(), viewScrollChangeEvent.scrollY()),
+                isItemShown)));
+  }
+
+  @Override public void removeBottomCardAnimation() {
+    View view = getPlaceHolder();
+    if (!bottomCardHidden && view != null) {
+      appCardLayout.animate()
```

Comment:
To improve code readability i would suggest you to extract this configuration to another method, possibly called configureAppCardAnimation

Reply:
Nice one, thanks. Fixed